docker/package/packages.py

```diff
@@ -431,6 +431,8 @@ def mk_node_unit(
         target_networks=networks.keys(),
         network_protos=networks_protos,
         meta=packages_meta,
+        additional_native_deps=[f"tezos-baker-{proto}" for proto in active_protocols]
```

Comment:
I think it would be better to lowercase `proto` here instead of `model.py`.

It would be done once instead of twice and we don't need to lowercase `tezos-node` nor `acl` (also maybe in the future we'll need to add more)

Reply:
we can't lowercase it here, `fedora` packages not lowercased

Reply:
since we lowercase only `ubuntu` packages, lowercasing is always safe even if there is no need to